src/cpp/monitor/include/monitor/events/Event.hpp

```diff
@@ -46,48 +47,46 @@ enum EventScope
 #define kSessionAdminSuspend     2006
 #define kSessionAdminTerminate   2007
 
-class Event
+#define kMaxEventDataSize     32
```

Comment:
Can we please document with a comment why we choose 32 here? (If I understand correctly this is because it's the maximum username length on Linux.)